packages/actor-query-operation-orderby-direct/lib/ActorQueryOperationOrderByDirect.ts

```diff
@@ -1,10 +1,14 @@
-import {SparqlExpressionEvaluator} from "@comunica/actor-query-operation-filter-direct";
-import {ActorQueryOperation, ActorQueryOperationTypedMediated,
-  IActorQueryOperationOutputBindings, IActorQueryOperationTypedMediatedArgs} from "@comunica/bus-query-operation";
-import {ActionContext, IActorTest} from "@comunica/core";
-import {termToString} from "rdf-string";
-import {Algebra} from "sparqlalgebrajs";
-import {SortIterator} from "./SortIterator";
+import { Term } from "rdf-js";
+import { termToString } from "rdf-string";
+import { Algebra } from "sparqlalgebrajs";
+import { AsyncEvaluator, isExpressionError } from 'sparqlee';
```

Comment:
The reason this class is called orderby-*direct* is because it makes use of actor-query-operation-filter-*direct*. Might make more sense then to create a new orderby actor instead here?

Reply:
OrderBySparqlee?

Reply:
Yea that's consistent with the others